can_tools/scrapers/official/DE/de_vaccine.py

```diff
@@ -4,6 +4,7 @@
 from bs4 import BeautifulSoup
```

Comment:
Delaware restructured their pages (moved all of the demographic charts onto one page) so this updates the scraper to match